Assets/MixedRealityToolkit/Attributes/ImplementsAttribute.cs

```diff
@@ -1,17 +1,17 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using Microsoft.MixedReality.Toolkit.Core.Definitions.Utilities;
+using Microsoft.MixedReality.Toolkit.Utilities;
 #if WINDOWS_UWP && !ENABLE_IL2CPP
-using Microsoft.MixedReality.Toolkit.Core.Extensions;
+using Microsoft.MixedReality.Toolkit;
 #endif // WINDOWS_UWP && !ENABLE_IL2CPP
 using System;
 
-namespace Microsoft.MixedReality.Toolkit.Core.Attributes
+namespace Microsoft.MixedReality.Toolkit
 {
     /// <summary>
     /// Constraint that allows selection of classes that implement a specific interface
-    /// when selecting a <see cref="Microsoft.MixedReality.Toolkit.Core.Definitions.Utilities.SystemType"/> with the Unity inspector.
+    /// when selecting a <see cref="SystemType"/> with the Unity inspector.
```

Comment:
>SystemType [](start = 36, length = 10)

do crefs not need namespaces if said namespaces are included?

Reply:
I don't think so, because of the namespace above. This was the VS simplify usings etc

---
In reply to: [267118065](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3658#discussion_r267118065) [](ancestors = 267118065)

Reply:
Confirmed VS doesn't require fully qualified class with namespace here because of the the class being within current namespace.

---
In reply to: [267119087](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3658#discussion_r267119087) [](ancestors = 267119087,267118065)